### ISSUE TITLE
PendingDeprecationWarning: the imp module is deprecated in favour of importlib

### ISSUE THREAD

**Issue by nedbat** [coveragepy/coveragepy#305]
*Originally reported by* **hugovk (Bitbucket: [hugovk](https://bitbucket.org/hugovk), GitHub: [hugovk](https://github.com/hugovk))**

----------------------------------------

How to reproduce:
```
import warnings
warnings.simplefilter('default')
import coverage
```

Actual result on Python 3.4:
```
/usr/lib/python3.4/imp.py:32: PendingDeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  PendingDeprecationWarning)
```

No warning on Python 2.6, 2.7, 3.2, 3.3 or PyPy.

Expected result: No warnings.

This happens when running tests for the Pillow project on Python 3.4 on Travis CI. For example: https://travis-ci.org/python-imaging/Pillow/jobs/24084189
  
(It just so happens that Pillow's tests are set up so warnings cause failures, but we'll ignore them for `import coverage` for the time-being.)

----------------------------------------
- Bitbucket: https://bitbucket.org/ned/coveragepy/issue/305

**Comment by nedbat** [comment]
Fixed in the commits leading to [aa1d5d23ca07 (bb)](https://bitbucket.org/ned/coveragepy/commits/aa1d5d23ca07)

**Comment by nedbat** [comment]
*Original comment by* **hugovk (Bitbucket: [hugovk](https://bitbucket.org/hugovk), GitHub: [hugovk](https://github.com/hugovk))**

----------------------------------------

Thanks!
